tests/test_client.py

```diff
@@ -876,5 +876,9 @@ def test_get_national_summary_votes_estimates(model_client, va_governor_county_d
 
     current = model_client.get_national_summary_votes_estimates(None, 0, [0.99])
 
-    pd.testing.assert_frame_equal(current, model_client.results_handler.final_results["nat_sum_data"])
-    pd.testing.assert_frame_equal(expected_df, model_client.results_handler.final_results["nat_sum_data"])
+    pd.testing.assert_frame_equal(
+        current, model_client.results_handler.final_results["nat_sum_data"], check_dtype=False
+    )
+    pd.testing.assert_frame_equal(
+        expected_df, model_client.results_handler.final_results["nat_sum_data"], check_dtype=False
```

Comment:
So I noticed the addition of `check_dtype=False` here and in every open PR right now, and I can confirm that it's necessary with the latest version of `pandas`.  It seems like `pandas` is now more strict with regard to data types.  Previously you could merge `int` (32 and 64) on `float` (32 and 64) just fine, but now this merge fails due to `agg_pred` being `float64` in the `current` and `expected_df` dataframes and `float64` in `model_client.results_handler.final_results["nat_sum_data"]`.  It should be an easy fix though, but not a huge deal/priority 🤔

Reply:
thanks for looking into this!